cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -77,6 +77,26 @@
     {% set show_categories = cat_controls.show_preview_categories if cat_controls is defined else true %}
     <article class="o-post-preview">
         <div class="m-meta-header">
+            {% if post.categories.exists() and show_categories %}
```

Comment:
TRIVIAL: If this is swapped around and `show_categories` is checked first it could bail out without needing to walk the object chain inside `post`.